pkg/cli/cmd/ls/ls.go

```diff
@@ -199,12 +201,16 @@ func printNotes(ctx context.DnoteCtx, bookName string) error {
 	for _, info := range infos {
 		body, isExcerpt := formatBody(info.Body)
 
-		rowid := log.ColorYellow.Sprintf("(%d)", info.RowID)
+		preface := log.ColorYellow.Sprintf("(%d)", info.RowID)
+		if timestamps {
+			local_time := time.Unix(0, info.AddedOn).Format(time.DateTime)
+			preface = log.ColorYellow.Sprintf("(%s)", local_time)
+		}
 		if isExcerpt {
 			body = fmt.Sprintf("%s %s", body, log.ColorYellow.Sprintf("[---More---]"))
 		}
 
```

Comment:
The logic for formatting the note content seems to be getting more complicated. What do you think about creating a separate function, and unit testing?

Reply:
I agree on both fronts. I went ahead and created a separate print note line function like what's there for printing book lines. I also added some tests that execute a view on notes. I added 3 tests which may be overkill. Let me know if that's too much.